prow/github/report/report.go

```diff
@@ -143,7 +143,10 @@ func Report(ghc GitHubClient, reportTemplate *template.Template, pj prowapi.Prow
 		return nil
 	}
 
-	if err := reportStatus(ghc, pj); err != nil {
+	if err := reportStatus(ghc, pj); err != nil &&
+		!strings.Contains(err.Error(), "\"message\":\"Not Found\"") {
```

Comment:
That error would also occur when the bot doesn't have access, for example because its a private repo. Swallowing the error makes a lot of sense, but can we maybe print something on `debug` level when this happens?

Reply:
Also, could you add a test for this? Either here or in prow/crier/reporters/github/reporter_test.go . The latter already has a test for a similiar scenario

Reply:
Make sense, both. Done